src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -190,6 +190,17 @@ internal T WithUserTokenCacheInternalForTest(ITokenCacheInternal tokenCacheInter
             return (T)this;
         }
 
+        /// <summary>
+        /// Enables or disables ADAL cache.
+        /// </summary>
+        /// <param name="disableAdalCache">Flag that indicates whether the ADAL cache operations will be invoked.</param>
```

Comment:
Maybe write something about that the fact that ADAL is the previous generation of authentication library and it's only needed if one migrates from ADAL.NET to MSAL.NET, and therefore `WithDisabledAdalCache(false)` is only for specific side by side scenarios and this has a performance impact on Token cache access/serialization/deserialization

Reply:
Updated.